sympy/calculus/singularities.py

```diff
@@ -57,18 +58,30 @@ def is_increasing(f, interval=S.Reals):
     False
     >>> is_increasing(4*x**3 - 6*x**2 - 72*x + 30, Interval(-2, 3))
     False
+    >>> is_increasing(x**2 + y, Interval(1, 2), symbol=x)
+    True
 
     """
-    if len(f.free_symbols) > 1:
-        raise NotImplementedError('is_increasing has not yet been implemented '
-                                  'for multivariate expressions')
-    symbol = f.free_symbols.pop()
+    f = sympify(f)
+    free_sym = f.free_symbols
+    if not sym:
+        if len(free_sym) > 1:
+            raise NotImplementedError('is_increasing has not yet been implemented '
+                                    'for multivariate expressions')
+        if len(free_sym) == 1:
+            symbol = free_sym.pop()
+        else:
+            return True
+
+    if sym:
+        symbol = sym.popitem()[1]
```

Comment:
Thanks. Though i didn't saw you code before, seeing `symbol=None` thing. I saw this after pushing. 
> if not symbol:
+            return False

For this line in your code this should ` return True` for `is_increasing` and `return False` for `is_strictly_increasing` and similar for `decreasing` one's.